core/js/src/main/scala/cats/effect/IORuntimeCompanionPlatform.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.effect
+
+import scala.concurrent.duration.FiniteDuration
+import scala.scalajs.concurrent.JSExecutionContext
+import scala.scalajs.js.timers
+
+private[effect] abstract class IORuntimeCompanionPlatform { self: IORuntime.type =>
+  lazy val default: IORuntime = {
```

Comment:
Not sure we get anything out of the laziness here, since `IORuntime` is already lazy, but I'm kind of picking nits.

Reply:
If not lazy, then calling `IORuntime.apply(myEc, myTimer)` will initialize this val.